projects/github-actions/test-results-to-slack/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Send tests results to Slack"
+description: "Send Slack notifications with test results."
+branding:
+  icon: 'activity'
+  color: 'green'
+inputs:
+  github_token:
+    description: "GitHub access token"
+    required: true
+    default: ${{ github.token }}
+  slack_token:
+    description: "Slack Bot access token"
+    required: true
+    default: ""
+  slack_channel:
```

Comment:
```suggestion
  slack_e2e_channel:
```

Reply:
As mentioned above, the GitHub action is not aware of the type of tests.